installers/MacInstaller/CMakeDMGSetup.scpt

```diff
@@ -41,8 +41,6 @@ on run argv
         set the bounds to { 400, 000, 900, 710 }
         set position of item "MantidPlot.app" to { 110, 220 }
         set position of item "Applications" to { 380, 220 }
-        set position of item "MantidPython (optional)" to { 380, 400 }
-        set position of item "MantidNotebook (optional)" to { 110, 400 }
         set position of item "MantidWorkbench" to { 110, 580 }
```

Comment:
Could we move the position so there's not so much whitespace. Something like: `set position of item "MantidWorkbench" to { 110, 400 }`